.tools/test/SPECIFICATION.md

```diff
@@ -0,0 +1,44 @@
+# Deployment Instructions
+These instructions assume a "target account" (where tests will be invoked) of "python" per [this repository's configuration](config/targets.yaml).
+You can replace Python with any of the other languages listed in this repository's configuration.
+
+To request an alternate configuration for your own repository, please [submit an issue](https://github.com/awsdocs/aws-doc-sdk-examples/issues/new?labels=type%2Fenhancement&labels=Tools&title=%5BEnhancement%5D%3A+Weathertop+Customization+Request&&) with the `Tools` label.
+
+## 1. Deploy Consumer Stack for your language (e.g. Python)
+Admin will:
+1. Set Python virtualenv within [sqs_lambda_to_batch_fargate directory](sqs_lambda_to_batch_fargate).
+1. `export LANAUGE_NAME=python`.
+1. Get Python account tokens.
+2. `cdk bootstrap` and `cdk deploy`.
+
+## 2. Enable Consumer Stack to receive event notifications
+User will:
+1. Set `status` to `enabled` in [targets.yaml](config/targets.yaml) for your language
+1. Raise PR.
+
+Admin will:
+1. Approve and merge PR.
+1. Set Python virtualenv within [eventbridge_rule_with_sns_fanout directory](eventbridge_rule_with_sns_fanout).
+1. Get Admin account tokens.
+1. `cdk bootstrap` and `cdk deploy`.
+1. Request user [submit test job](#3-submit-test-job).
+
+## 3. Submit test job
```

Comment:
Can we make a script to do this?

Reply:
I had the same thought as I was testing! It's definitely possible, though you'd need to string together a few SDK calls to get the job def, compute env, and queue details from the target stack, then add some defaults into a submit job command.

Reply:
Nebulous nice to have backlog, then :)